analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.PatternMatching.cs

```diff
@@ -384,11 +384,11 @@ public void DeclarationPattern_Var_Deconstruction_DoesNotSetConstraint()
             ValidateSetBoolConstraint(isPattern, OperationKindEx.DeclarationPattern, expectedBoolConstraint);
 
         [DataTestMethod]
-        [DataRow("objectUnknown", "is object o", "NotNull", "Null")]
-        [DataRow("exceptionUnknown", "is object { }", "NotNull", "Null")]
-        [DataRow("exceptionUnknown", "is Exception { }", "NotNull", "Null")]
-        [DataRow("exceptionUnknown", "is FormatException { }", "NotNull", null)]
-        public void DeclarationPatternSetBoolConstraint_TwoStates(string testedSymbol, string isPattern, string expectedForTrue, string expectedForFalse) =>
+        [DataRow("objectUnknown", "is object o", new[] { ConstraintKind.ObjectNotNull }, new[] { ConstraintKind.ObjectNull })]
+        [DataRow("exceptionUnknown", "is object { }", new[] { ConstraintKind.ObjectNotNull }, new[] { ConstraintKind.ObjectNull })]
+        [DataRow("exceptionUnknown", "is Exception { }", new[] { ConstraintKind.ObjectNotNull }, new[] { ConstraintKind.ObjectNull })]
+        [DataRow("exceptionUnknown", "is FormatException { }", new[] { ConstraintKind.ObjectNotNull }, null)]
+        public void DeclarationPatternSetBoolConstraint_TwoStates(string testedSymbol, string isPattern, ConstraintKind[] expectedForTrue, ConstraintKind[] expectedForFalse) =>
```

Comment:
`expectedForTrue` never changes -> let's hardcode it to the body
```suggestion
        public void DeclarationPatternSetBoolConstraint_TwoStates(string testedSymbol, string isPattern, ConstraintKind[] expectedForFalse) =>
```